packages/algorand-js/test/mocks/account.ts

```diff
@@ -0,0 +1,11 @@
+export const johnAccount = {
+  addr: '2UBZKFR6RCZL7R24ZG327VKPTPJUPFM6WTG7PJG2ZJLU234F5RGXFLTAKA',
```

Comment:
is it possible to use addresses from `config` files ? maybe using mnenomic phrase ?

Reply:
I think this is fine. In algob also we use mock accounts as `{ addr: .., sk: ..}`, as it's type is `AccountSDK`. Did you mean something else?